File: samples/server/openapi3/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/model/InlineObject.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*
import io.swagger.annotations.ApiModelProperty

/**
 * 
 * @param name Updated name of the pet
 * @param status Updated status of the pet
 */
data class InlineObject (

        @ApiModelProperty(example = "null", value = "Updated name of the pet")
        @JsonProperty("name") val name: String? = null,

        @ApiModelProperty(example = "null", value = "Updated status of the pet")
        @JsonProperty("status") val status: String? = null
) {

}

File: samples/client/petstore/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject.scala

package org.openapitools.client.model


case class InlineObject (
    /* Updated name of the pet */
    _name: Option[String],
    /* Updated status of the pet */
    _status: Option[String]
)
object InlineObject {
    def toStringBody(var_name: Object, var_status: Object) =
        s"""
        | {
        | "name":$var_name,"status":$var_status
        | }
        """.stripMargin
}
